examples/nl-core-Vaccination-03-Location-01.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Where do these examples come from? Have they been manually created? They raise a lot of errors BTW.

Reply:
Yes they have been manually created and just the first draft.